### ISSUE TITLE
Server gets down after several thousands of records  written using http API.

### ISSUE THREAD

**Issue by lqju** [taosdata/TDengine#1232]
version: 1.6.4.5 

使用http接口，执行自动建表语句插入数据（INSERT INTO test.t_206e3219d3de1f3180c683587e4179a2 USING apm.web_page_cost_time_info TAGS () values ()）。写入几千条后，taosd进程挂掉了。

taosdlog：

```02/17 06:47:35.841429 7f5f62d33700 MND msg:create is sent to dnode
02/17 06:47:35.841436 7f5f62d33700 MND meter:0.apm.t_4e247fa6120eb56bb3b7d69a1f74a208 is automatically created by root from 0.apm.web_page_cost_time_info, code:0
02/17 06:47:35.841451 7f5f62d33700 MND 0.apm.t_4e247fa6120eb56bb3b7d69a1f74a208, uid:6597099719111 meter meta is retrieved
02/17 06:47:35.859053 7f5f69540700 MND meter:0.apm.t_598066e1726a820f1eb1025fdf11b16d auto created task added
02/17 06:47:35.859144 7f5f62d33700 MND meter:0.apm.t_598066e1726a820f1eb1025fdf11b16d, pConnDb:0x7f5f20000d00, pConnDbName:0.apm, pMeterDb:0x7f5f20000d00, pMeterDbName:0.apm
02/17 06:47:35.859175 7f5f62d33700 MND table:0.apm.t_598066e1726a820f1eb1025fdf11b16d, create table in vgroup, vgId:6 sid:458 vnode:5 uid:6597099784648 db:0.apm
02/17 06:47:35.859288 7f5f62d33700 MND-SDB table:meters, a record is inserted:0.apm.t_598066e1726a820f1eb1025fdf11b16d, sdbVersion:2505 id:2492 rowSize:623 numOfRows:2465 fileSize:1605277
02/17 06:47:35.859306 7f5f62d33700 MND table:0.apm.t_598066e1726a820f1eb1025fdf11b16d, send create table msg to dnode, vgId:6, sid:458, vnode:5
02/17 06:47:35.859318 7f5f62d33700 MND msg:create is sent to dnode
02/17 06:47:35.859323 7f5f62d33700 MND meter:0.apm.t_598066e1726a820f1eb1025fdf11b16d is automatically created by root from 0.apm.web_page_cost_time_info, code:0
02/17 06:47:35.859331 7f5f62d33700 MND 0.apm.t_598066e1726a820f1eb1025fdf11b16d, uid:6597099784648 meter meta is retrieved
02/17 06:47:35.859761 7f5f6c546700 ERROR DND vid:5 sid:458, not active table
02/17 06:47:35.932504 7f5f6b544700 MND msg:meter-cfg is received from dnode
02/17 06:47:35.932539 7f5f6b544700 MND msg:meter-cfg-rsp is sent to dnode
02/17 06:47:35.932546 7f5f6b544700 MND msg:create-rsp is received from dnode
02/17 06:47:35.932551 7f5f6b544700 MND msg:create-rsp is received from dnode
02/17 06:47:35.942081 7f5f63d35700 MND 0.apm.t_598066e1726a820f1eb1025fdf11b16d, uid:6597099784648 meter meta is retrieved
```

使用taos shell执行相同的sql语句，一次性导入一百多万数据无异常。

**Comment by guanshengliang** [comment]
Sorry, no reply for long time
The latest 1.6.6.0 no longer has this problem
and the 2.0 version will release soon

**Comment by guanshengliang** [comment]
There is no response to this issue for a long time, and this issue has been fixed in 2.0 version, so close it
